### PR TITLE
Sonya test

### PR DESCRIPTION
**Pull request by BlueSuperGlue**
removed extra divs

### CODE REVIEW

**Comment by BlueSuperGlue** [comment]
this addresses issue #21

File: items/show.php [hunk 1]
```diff
@@ -21,9 +21,9 @@
                         <div class="col-12">
                             <?php if ($imagesCount === 1): ?>
                                 <!--  <img class="card-img-top" src="<?php //echo url('/'); ?>files/original/<?php //echo $image->filename; ?>" /> -->
-                                <?php echo file_markup($image,array('imageSize' => 'fullsize','linkToFile' => false,'imgAttributes' => array('class' => 'w-100','data-toggle'=>'#carouselExample', 'data-slide-to'=> strval($imagesCount-1) ))); ?>
```
Comment (apjmason) [inline items/show.php:24]: Not sure I agree with this decision. Can you revert to full size

File: functions.php [hunk 2]
```diff
@@ -9,7 +9,7 @@ function random_featured_exhibit()
     } else {
         $html .= '<p>' . __('You have no featured exhibits.') . '</p>';
     }
-    $html = apply_filters('exhibit_builder_display_random_featured_exhibit', $html);
```
Comment (apjmason) [inline functions.php:12]: Not sure why you removed this filter? Please restore

**Review by apjmason — CHANGES_REQUESTED** [review]
Hey the div fix looks good but there are a few extra changes that should be reverted. See code comments

**Pull request merged**